<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>50</y>
     <width>811</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Search Markers</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>42</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gadugi</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>First Cell type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>92</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gadugi</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Second Cell type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>142</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gadugi</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Thrid Cell type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="C1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <item>
     <property name="text">
      <string>Cell Type</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lymphocyte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Myeloid cell</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="C2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="C3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Done_btn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>186</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>290</y>
     <width>811</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Markers</string>
   </property>
   <widget class="QTableWidget" name="Show_Marker">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>31</y>
      <width>761</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Home_btn">
   <property name="geometry">
    <rect>
     <x>828</x>
     <y>17</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
